packages/components/progress-bar/stories/progress-bar.happo.js

```diff
@@ -1,26 +1,26 @@
-import { storiesOf } from '@open-wc/demoing-storybook';
-import '@tradeshift/elements';
-import '@tradeshift/elements.progress-bar';
-
-import { createHappoStories } from '../../../../.storybook-happo/utils';
-
-storiesOf('ts-progress-bar', module).add('test', () => {
-	const properties = {
-		done: {
-			'0': 0,
-			'5': 5,
-			'100': 100,
-			'600': 600
-		},
-		total: {
-			'10': 10,
-			'1000': 1000
-		}
-	};
-
-	const options = {
-		columns: 2
-	};
-
-	return createHappoStories('progress-bar', properties, '', options);
-});
+// import { storiesOf } from '@open-wc/demoing-storybook';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACommentedCode)